omniport/omniport/middleware/routes_control.py

```diff
@@ -42,4 +45,15 @@ def __call__(self, request):
 
         response = self.get_response(request)
```

Comment:
If you are checking over every app's URL, shouldn't you call the get_response function after checking everything there?